### ISSUE TITLE
Performance improvements in EF !?

### ISSUE THREAD

**Issue by ErikEJ** [aspnet/EntityFramework6#324]
This blog post: https://blogs.msdn.microsoft.com/dotnet/2017/07/24/net-framework-july-2017-preview-of-quality-rollup/  mentions "performance improvements in Entity Framework" - are you able to share what they are, and which EF version is affected - assume it is 5?

**Comment by divega** [comment]
I am quite sure the term "performance improvements" is being used to describe the added ability to control the query cache size in the versions of EF runtime that are included in .NET Framework (e.g. EF versions 4 to 5 can take advantage of this). 

This was originally implemented in a hotfix and it can be done in the configuration system, e.g.:

``` xml
  <appSettings>
    <add key="EntityFramework_QueryCacheSize" value="5000"/>
  </appSettings>
```

As you probably know, the equivalent setting already existed in EF6 in the EF configuration section:

``` xml
 <entityFramework> 
    <queryCache size='5000' cleaningIntervalInSeconds='120'/> 
 </entityFramework> 
```

**Comment by ErikEJ** [comment]
@divega Where is the EF6 setting documented?

**Comment by divega** [comment]
@ErikEJ I did a quick search and the only places I found it from us is in our original announcement blog post, and in the version history page at https://msdn.microsoft.com/en-us/library/jj574253(v=vs.113).aspx. The original commit https://github.com/aspnet/EntityFramework6/commit/528fc83bfb579af7c3c9145d5516062bca983b2f is now indexed too :smile_cat:

I have created a new issue to add a page about the configuration section: https://github.com/aspnet/EntityFramework.Docs/issues/426. We should be able to do this when the EF6 documentation gets migrated to docs.microsoft.com.
